portals/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx

```diff
@@ -472,6 +472,7 @@ class ApiConsole extends React.Component {
                         productionApiKey={productionApiKey}
                         sandboxApiKey={sandboxApiKey}
                         api={this.state.api}
+                        URLs={null}
```

Comment:
without passing null you can also set the defaultProp's value of URLs in TryoutController component and leave it empty here.

Reply:
ack. will address this from a seperate PR